aspnetcore/tutorials/signalr.md

```diff
@@ -224,7 +224,7 @@ The SignalR server must be configured to pass SignalR requests to SignalR.
 * In the integrated terminal, run the following command:
 
   ```console
-  dotnet run -p SignalRChat
+  dotnet run -p SignalRChat.csproj
```

Comment:
I *think* that, if there's only one `.csproj` in the directory you can omit the project parameter, but for completeness I agree we should include it in this example. 👍